src/core/lib/iomgr/event_engine/iomgr.h

```diff
@@ -17,8 +17,16 @@
 
 #include <grpc/event_engine/event_engine.h>
 
-// This can be called anywhere in the EE-based iomgr impl where we need to
-// access the global EE instance.
+/// This can be called anywhere in the EventEngine-based iomgr impl where we
+/// need to access the global EventEngine instance.
 grpc_event_engine::experimental::EventEngine* grpc_iomgr_event_engine();
 
+namespace grpc_core {
+
+/// Set the default EventEngine. This engine is shut down along with iomgr.
+void SetDefaultEventEngine(
+    std::unique_ptr<grpc_event_engine::experimental::EventEngine> event_engine);
```

Comment:
optional: If we are going to expose a certain EE setter in Core's surface API, we might want to just use a plain pointer here?

Reply:
The EE surface also contain unique_ptr, so we need to work on smart pointers in Cython anyway.

Reply:
For now, unique_ptr<> is fine.

I don't yet have a clear picture of what the long-term API will look like for setting the EE instance for a channel or server.  We'll figure that out later.

Reply:
SGTM, thanks!